src/autocomplete/popup.js

```diff
@@ -153,7 +153,7 @@ class AcePopup {
                 selected.setAttribute("aria-setsize", popup.data.length);
                 selected.setAttribute("aria-posinset", row + 1);
                 selected.setAttribute("aria-describedby", "doc-tooltip");
-                selected.setAttribute("aria-selected", "true");
+                selected.setAttribute(userAgent.isSafari ? "aria-current" : "aria-selected", "true");
```

Comment:
We might wanna do the `isSafari` check more above to group Safari related changes in one place.

Reply:
@sokomari refactored the PR a bit to make it clearer, do you think this is an improvement?